src/open_inwoner/scss/views/_view.scss

```
@@ -62,4 +62,14 @@
       display: inline;
     }
   }
+
+  // Remove Safari iOS phone-parsing from specific elements
+  .phone-parsing {
```

```suggestion
  .skip-phone-parsing {
```
or something like it so that the function of the class is clear from the name alone (otherwise you have to come hear to look at the comment)

---

Ah, yeah, good one, will do.